src/radius.c

```diff
@@ -690,22 +690,18 @@ int radius_timeout(struct radius_t *this) {
 
       if (RADIUS_QUEUE_HASPKT(this->queue[this->first].p)) {
 
-	/* Use the correct port for accounting and authentication */
-	if (RADIUS_QUEUE_PKT(this->queue[this->first].p,code)
-	    == RADIUS_CODE_ACCOUNTING_REQUEST)
-	  addr.sin_port = htons(this->acctport);
-	else
-	  addr.sin_port = htons(this->authport);
-
-	if (sendto(this->fd,
-		   RADIUS_QUEUE_PKTPTR(this->queue[this->first].p),
-		   ntohs(RADIUS_QUEUE_PKT(this->queue[this->first].p, length)),
-		   0, (struct sockaddr *) &addr, sizeof(addr)) < 0) {
+        /* Use the correct port for accounting and authentication */
+        if (RADIUS_QUEUE_PKT(this->queue[this->first].p,code)
+            == RADIUS_CODE_ACCOUNTING_REQUEST)
+          addr.sin_port = htons(this->acctport);
+	      else
```

Comment:
Thanks for response. 

I see that several files have mixed tab and spaces. What are preferred?